2 Patrol Outcomes + Patrol Art

SnapshotManor opened the pull request:
<!-- Write BELOW The Headers and ABOVE The comments else it may not be viewable. -->
<!-- You can view CONTRIBUTING.md for a detailed description of the pull request process. -->
<!-- Be sure to name your PR something descriptive and succinct; include Bugfix: Feature: Enhancement: or Content: in the title to describe what type of PR it is. -->

## About The Pull Request

- Adds two new outcomes for patrol ID "gen_bord_kittypet_friends_with_dog"
- Adds one new patrol art for "gen_bord_kittypet_friends_with_dog"

## Why This Is Good For ClanGen

I wanted to create art for Saema's recently added patrols, as well as add some of my own takes on outcomes.

## Proof of Testing

![1](https://github.com/user-attachments/assets/83464c09-a9d8-46b1-92dc-019d75ce69df)
![2bloodthirsty](https://github.com/user-attachments/assets/2969a790-9f5e-4712-bbeb-b5156dc1f204)

## Changelog/Credits

- Added two new patrol outcomes 
- Added one new patrol art

SnapshotManor committed:
2 New Patrol Outcomes + Changes

Added 2 new patrol outcomes as well as editing existing patrol outcomes for the patrol art I made.

SnapshotManor committed:
Add files via upload

Added 2 new patrol outcomes as well as editing existing patrol outcomes for the patrol art I made.

SnapshotManor committed:
Add files via upload

New patrol art

scribblecrumb commented on resources/dicts/patrols/general/border.json:
```diff
@@ -413,7 +413,7 @@
             },
             {
-                "text": "The patrol fights bravely, but you only need to be unlucky once. r_c gets taken out of the fight with one snap of the beast's huge jaws. The rest of the patrol rallies to defend {PRONOUN/r_c/object}, and r_c is safely helped back to camp, where the medicine cat den awaits.",
+                "text": "The patrol fights bravely, but you only need to be unlucky once. r_c gets taken out of the fight with one snap of the beast's huge jaws. The rest of the patrol rallies to defend {PRONOUN/r_c/object}, and r_c is safely helped back to camp, where the medicine den awaits.",
```
you've got a few changes in this file that seem to be reverting recent fixes.  you'll need to undo those!

scribblecrumb commented on resources/dicts/patrols/general/border.json:
```diff
@@ -2462,8 +2481,27 @@
+            "new_cat": [
+            ["kittypet", "meeting", "exists"]
+            ],
+            "relationships": [
```
Since you used the "exists" tag, there's the possibility for this event to occur with an outsider who has established relationships with the cats on the patrol.  I'd suggest adding a relationship change to reduce trust/respect/like/comfort as well.

scribblecrumb commented on resources/dicts/patrols/general/border.json:
```diff
@@ -2462,8 +2481,27 @@
+                        }
+            ],
+            "weight": 10,
+            "stat_trait": ["bloodthirsty"],
```
```suggestion
            "stat_trait": ["bloodthirsty", "vengeful"],
```
I think this is probably pretty fitting as well

scribblecrumb commented on resources/dicts/patrols/general/border.json:
```diff
@@ -2360,8 +2360,27 @@
+            "text": "p_l lashes out to swipe at the dog but is interrupted by a kittypet rushing to stand in {PRONOUN/p_l/poss} way. p_l's fur bristles for a moment, but the kittypet tells {PRONOUN/p_l/object} to calm down, that this is {PRONOUN/n_c:0/poss} friend. As p_l recovers from.. whatever this is, {PRONOUN/p_l/subject} offers the kittypet a place to stay in c_n- this kittypet's skill in c ...
+            "exp": 10,
+	    "new_cat": [
+		["status:warrior", "backstory:kittypet1"]
```
```suggestion
		["status:warrior", "backstory:kittypet1", "kittypet", "exists"]
```
up to you if you accept this suggestion! but adding these would allow for existing kittypets to be acquired via this event.

scribblecrumb commented on resources/dicts/patrols/general/border.json:
```diff
@@ -2462,8 +2481,27 @@
+            "art": "gen_kittypet_friends_with_dog1"
+	},
+    {
+        "text": "p_l lets out a vicious yowl, ignoring the now shrill voice calling out to {PRONOUN/p_l/object}, pleading with her to stop as {PRONOUN/p_l/subject} {VERB/p_l/attack/attacks} the dog, violently tearing at the dog's pelt as it yaps and whines. After the tussle, the dog hauls itself towards the now teary-eyed kittypet, and p_l can't help but smirk at {PRONOUN/n_c:0/poss} terrified ex ...
```
```suggestion
        "text": "p_l lets out a vicious yowl, ignoring the now shrill voice calling out to {PRONOUN/p_l/object}, pleading with {PRONOUN/p_l/object} to stop as {PRONOUN/p_l/subject} {VERB/p_l/attack/attacks} the dog, violently tearing at the dog's pelt as it yaps and whines. After the tussle, the dog hauls itself towards the now teary-eyed kittypet, and p_l can't help but smirk at {PRONOUN/n_c:0/poss} terrified expression.",
```
Also, do the existing outcomes refer to the dog with they/them or with its/it pronouns?  Double check to ensure it's consistent.

SnapshotManor committed:
Add files via upload

SnapshotManor committed:
Missed Art Change

Forgot to edit the patrol art for the existing patrols when I made the previous changes

anjubatus reviewed (changes requested):
Hi! I have some changes that I'd like to see in the art you submitted. Most important being the head and ears of the cat being a bit odd shape, but I have some other minor adjustments too, including removing some unnecessary details from the dog's face.
![Näyttökuva 2024-09-22 141554](https://github.com/user-attachments/assets/4b540264-3b80-40fb-ba8d-50432c569d3f)
![Näyttökuva 2024-09-22 141609](https://github.com/user-attachments/assets/aa9681af-3d5f-497d-95e5-f796c30bc36d)

scribblecrumb commented on resources/dicts/patrols/general/border.json:
```diff
@@ -2317,7 +2317,7 @@
     "season": ["any"],
     "types": ["border"],
-    "tags": ["new_cat"],
+    "tags": [],
```
```suggestion
    "tags": ["new_cat],
```
this tag needs to stay!

scribblecrumb commented on resources/dicts/patrols/general/border.json:
```diff
@@ -2484,10 +2484,10 @@
 	},
     {
-        "text": "p_l lets out a vicious yowl, ignoring the now shrill voice calling out to {PRONOUN/p_l/object}, pleading with her to stop as {PRONOUN/p_l/subject} {VERB/p_l/attack/attacks} the dog, violently tearing at the dog's pelt as it yaps and whines. After the tussle, the dog hauls itself towards the now teary-eyed kittypet, and p_l can't help but smirk at {PRONOUN/n_c:0/poss} terrified ex ...
+        "text": "p_l lets out a vicious yowl, ignoring the now shrill voice calling out to {PRONOUN/p_l/object}, pleading with {PRONOUN/p_l/object} to stop as {PRONOUN/p_l/subject} {VERB/p_l/attack/attacks} the dog, violently tearing at the dog's pelt as it yaps and whines. After the tussle, the dog hauls itself towards the now teary-eyed kittypet, and p_l can't help but smirk at {PRONOUN/n_c:0/p ...
```
```suggestion
        "text": "p_l lets out a vicious yowl, ignoring the now shrill voice calling out to {PRONOUN/p_l/object}, pleading with {PRONOUN/p_l/object} to stop as {PRONOUN/p_l/subject} {VERB/p_l/attack/attacks} the dog, violently tearing at the dog's pelt as they yap and whine. After the tussle, the dog hauls themself towards the now teary-eyed kittypet, and p_l can't help but smirk at {PRONOUN/n_c:0/poss} terrified expression.",
```
Original outcomes used they/them pronouns for the dog, so I've adjusted this accordingly.

SnapshotManor committed:
Update resources/dicts/patrols/general/border.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

SnapshotManor committed:
Update resources/dicts/patrols/general/border.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

SnapshotManor committed:
Art Changes 4 Anju

scribblecrumb commented:
![image](https://github.com/user-attachments/assets/f68d2b64-5ab3-4ac5-9030-a40e3914cdc6)

Some notes on the dog (from Kori)

![image](https://github.com/user-attachments/assets/16ce9cc9-6b4a-4cc2-b4ab-89326bb21cb3)

and a suggestion from Tiny, mostly just regarding the collar.  It was also suggested that you might consider adding a tag or buckle to the cat collar, as well as consider giving the dog a collar to better visually connect the two.

As I'm sure you can tell, we're a bit strict on patrol art quality, but you're doing very well!  Once we have these last few bits ironed out, I think we'll be good.

SnapshotManor committed:
Additional Art Changes

For Kori & Tiny!

scribblecrumb commented:
You've got a failing json test! I'd suggest running the json through a json checker to see what's missing

SnapshotManor committed:
Json Fix

No idea how it got messed up but! I fixed it! lol

scribblecrumb commented:
![image](https://github.com/user-attachments/assets/9bf00e25-0399-4ef6-86eb-fe290e520e39)

One last bit of feedback from Anju

SnapshotManor committed:
4 Anju

My apologies- I didn't even see the lines removed in your first request, totally missed it. Fixed now! c:
